tracer/build/_build/Build.Steps.cs

```diff
@@ -1029,7 +1007,7 @@ void ExtractDebugInfoAndStripSymbols()
                     //.WithMemoryDumpAfter(timeoutInMinutes: 30)
                     .EnableNoRestore()
                     .EnableNoBuild()
-                    .SetProcessEnvironmentVariable("TracerHomeDirectory", TracerHomeDirectory)
+                    .SetProcessEnvironmentVariable("TracerHomeDirectory", MonitoringHomeDirectory)
```

Comment:
Won't this cause some confusion? In other places in this PR, it seemed like we are deprecating the concept of a "TracerHome", but maybe I misinterpreted?

Reply:
> Won't this cause some confusion? In other places in this PR, it seemed like we are deprecating the concept of a "TracerHome", but maybe I misinterpreted?

Yep, we're deprecating the use of `TracerHome`, there's still some cleanup to do to remove all traces of it though 🙂 I'll push that to a later PR for simplicity